Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10-slim as build-stage
+
+WORKDIR /tmp/code
+
+COPY . .
+
+RUN pip wheel --wheel-dir ./dist .
+
+FROM python:3.10-slim
+
+WORKDIR /app
+
+COPY --from=build-stage /tmp/code/dist .
+
+RUN pip install --no-cache-dir --no-index --find-links . bloodhound-tools
```

Comment:
Since this is an artifact of your pythoncookie template either:

1) Resubmit this Dockerfile along with the changes made using your template w/ additional stages that have ENTRYPOINTS for each program

2) Extend Dockerfile to include ENTRYPOINTS for each program